docs/pages/build-reference/variables.mdx

```diff
@@ -95,27 +97,26 @@ To create **app-specific secrets**, navigate to [the "Secrets" tab in your proje
 To create a new secret, run `eas secret:create`
 
 ```
```

Comment:
Outside of the scope of these changes, but should we use `Terminal` here instead of a code block?

Reply:
It's a good idea, we can adjust that in the separate PR.

Reply:
The `Terminal` component is buggy:

<img width="1317" alt="Screenshot 2022-10-05 at 15 01 08" src="https://user-images.githubusercontent.com/5256730/194070820-2c1167b6-52b6-446e-84e1-880788407fd0.png">
<img width="927" alt="Screenshot 2022-10-05 at 15 01 05" src="https://user-images.githubusercontent.com/5256730/194070824-bf2897ab-7440-4839-8f52-3fb6ad369370.png">

Feel free to change that in a separate PR.

Reply:
Nice catch, we are missing `white-space: pre` in the Terminal `codeStyle`. 

Will fix this in the following PR and move the table into Terminal.